tests/attributes/src/lib.rs

```diff
@@ -4,7 +4,7 @@ use hax_lib_macros as hax;
 const u32_max: u32 = 90000;
 
 #[hax::requires(x > 10 && y > 10 && z > 10 && x + y + z < u32_max)]
-#[hax::ensures(|result| result > 32)]
+#[hax::ensures(|result| hax_lib::implies(true, || result > 32))]
```

Comment:
IS the F* code for this checked in?

Reply:
I added a snapshot this test crate: https://github.com/hacspec/hax/pull/388/commits/de3ed5fffb39ee91c7751c027afdd9b1e993f480